common/src/main/java/fr/openent/form/core/models/Question.java

```diff
@@ -1,136 +1,160 @@
+        this.sectionPosition = sectionPosition;
         return this;
     }
-    public Question setNextSectionId(Number nextSectionId) {
-        this.nextSectionId = nextSectionId;
+
+    public Question setConditional(Boolean conditional) {
+        this.conditional = conditional;
         return this;
     }
-    public Question setPosition(Number position) {
-        this.position = position;
+
+    public Question setPlaceholder(String placeholder) {
+        this.placeholder = placeholder;
         return this;
     }
-    public Question setQuestionId(Number questionId) {
-        this.questionId = questionId;
+
+    public Question setMatrixId(Number matrixId) {
+        this.matrixId = matrixId;
         return this;
     }
-    public Question setQuestionType(Integer questionType) {
-        this.questionType = questionType;
+
+    public Question setMatrixPosition(Number matrixPosition) {
+        this.matrixPosition = matrixPosition;
         return this;
     }
-    public Question setType(ChoiceTypes type) {
-        this.type = type;
+
+    public Question setQuestionChoices(List<QuestionChoice> questionChoices) {
+        this.questionChoices = questionChoices;
         return this;
     }
-    public Question setValue(String value) {
-        this.value = value;
+
+    public Question setChildren(List<Question> children) {
+        this.children = children;
         return this;
     }
 
+
     // Functions
 
     public JsonObject toJson() {
         return new JsonObject()
-                .put(Fields.QUESTION_TYPE, this.questionType)
-                .put(Fields.ID, this.id)
-                .put(Fields.QUESTION_ID, this.questionId)
-                .put(Fields.VALUE, this.value)
-                .put(Fields.POSITION, this.position)
-                .put(Fields.NEXT_SECTION_ID, this.nextSectionId)
-                .put(Fields.NB_RESPONSES, this.nbResponses)
-                .put(Fields.TYPE, this.type)
-                .put(Fields.CODE, this.code)
-                .put(Fields.NAME, this.name)
-                .put(Fields.MATRIX_ID, this.matrixId)
-                .put(Fields.MATRIX_POSITION, this.matrixPosition);
+                .put(ID, this.id)
+                .put(FORM_ID, this.formId)
+                .put(TITLE, this.title)
+                .put(POSITION, this.position)
+                .put(QUESTION_TYPE, this.questionType)
+                .put(STATEMENT, this.statement)
+                .put(MANDATORY, this.mandatory)
+                .put(ORIGINAL_QUESTION_ID, this.originalQuestionId)
+                .put(SECTION_ID, this.sectionId)
+                .put(SECTION_POSITION, this.sectionPosition)
+                .put(CONDITIONAL, this.conditional)
+                .put(PLACEHOLDER, this.placeholder)
+                .put(MATRIX_ID, this.matrixId)
+                .put(MATRIX_POSITION, this.matrixPosition)
+                .put(QUESTION_CHOICES, new JsonArray(this.questionChoices))
```

Comment:
un peut dangereux ici:
this.questionChoices est de type QuestionChoice (et non json object), il faut  garantir que tu convertis bien l'élément en jsonObject:

```suggestion
// Dans Model.java :
default JsonArray toArray(List<I> models) {
        return new JsonArray(models.stream().map(Model::toJson).collect(Collectors.toList()));
    }
    
// ici, dans ta méthode toJson():
if (questionChoices != null) result.put(QUESTION_CHOICES, new QuestionChoice().toArray(questionChoices));
```